### ISSUE TITLE
CFPGan not doing anything? 

### ISSUE THREAD

**Issue by altryne** [hlky/stable-diffusion-webui#113]
### with CFPGan
![00204-523150267_cartoon_dog_,__unity_3d_,__still_frame_from_rick_and_morty-gfpgan](https://user-images.githubusercontent.com/463317/187022809-f5deb2ee-18b1-4837-9abe-d450c2b7ba64.png)
### without
![00204-523150267_cartoon_dog_,__unity_3d_,__still_frame_from_rick_and_morty](https://user-images.githubusercontent.com/463317/187022810-8c6abd89-52b2-403a-a0d4-fc9a0e9fa2a8.png)
### with CFPGan
![00206-523150267_cartoon_dog_,__by_greg_rutkowski_,__unity_3d_,__still_frame_from_rick_and_morty-gfpgan](https://user-images.githubusercontent.com/463317/187022811-7e476f25-e4ab-40c7-a626-51286ec67d3b.png)
### without
![00206-523150267_cartoon_dog_,__by_greg_rutkowski_,__unity_3d_,__still_frame_from_rick_and_morty](https://user-images.githubusercontent.com/463317/187022813-45552aa5-fd65-4675-9da3-7d72f147dbae.png)

**Comment by EyeDeck** [comment]
It typically only triggers on something approaching a photorealistic human face.

**Comment by parsec501** [comment]
GFPGAN is a face upscaler, for human faces. You are looking for BSRGAN/SWINIR/RealESRGAN.

**Comment by jtm732** [comment]
I'm aware this was closed, but I ran a test with a human face and had both:

"Fix faces using GFPGAN
Upscale images using RealESRGAN"

Checked off, and the result showed that both were applied (supposedly). I noticed the output had some quite distorted human faces, and I put one into the GFPGAN section of the WebUI, and the output was amazingly better. I'm thinking it isn't actually being applied for some reason. I am not sure if REALESRGAN is also being applied, but I will do some tests to find out.
